src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/ReferenceDataController.kt

```diff
@@ -8,15 +8,15 @@ import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.pecs.jpc.service.MapFriendlyLocationService
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.LocationsService
 
 @RestController
-@RequestMapping(name = "Locations", path = ["/locations"], produces = [MediaType.APPLICATION_JSON_VALUE])
-class LocationsController(@Autowired val locationService: MapFriendlyLocationService) {
+@RequestMapping(name = "Reference Data", path = ["/reference"], produces = [MediaType.APPLICATION_JSON_VALUE])
+class ReferenceDataController(@Autowired val locationService: LocationsService) {
```

Comment:
Renamed to avoid conflict/confusion with other controllers.  ReferenceDataController seemed reasonable to me.